.gitlab-ci.yml

```diff
@@ -86,6 +86,7 @@ build_preview:
     - make dependencies
     - make config
     - ./node_modules/.bin/jest --testPathPattern=assets/scripts/tests/
+    - git config --global url."https://${GITHUB_TOKEN}@github.com/DataDog".insteadOf https://github.com/DataDog
```

Comment:
can we put this in `before_script` instead? i imagine both `build_preview` and `build_live` will need it